packages/terra-demographics-banner/src/_SmallDemographicsBannerDisplay.jsx

```diff
@@ -30,9 +30,8 @@ export default (props) => {
   } = props;
 
   const mainClasses = cx([
-    'demographics-banner',
```

Comment:
The regular demographics banner has lost styles:

PR Deployment:

![image](https://user-images.githubusercontent.com/6720991/76788533-1a70c500-6789-11ea-9ab7-1a674d07ef81.png)

Prod terra-ui:

![image](https://user-images.githubusercontent.com/6720991/76788569-2d839500-6789-11ea-88d4-84e248faade1.png)

Reply:
[Added `demographics-banner` class. ](https://github.com/cerner/terra-core/pull/2856/commits/29442b337f5a3a0d8f21da8778dc7e6392406055#diff-a2e362c616f1393052c0242d56c95209)